jest.setup.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+configure({ adapter: new Adapter() });
```

Comment:
We should remove this file since this is specific to React and Jest can be used for any JS project, not just React.

Reply:
We could make it similar to our babel config where we have a `react` option?

Reply:
I'm unsure how i could make it an option, as the package must be passed as a string to the `preset` option. 

If this package must agnostic to React applications, then i agree with @ivolimasilva, that this should not be a concern of this package and should be delegated to their own configs.

Reply:
@satazor to make it like that, than this would be a _config_ and not a _preset_. It's more powerful but it's that the goal of this project?

Reply:
@threequartersjohn it seems there's no way, unless we make this package a config rather than a preset.

The usage would be similar to babel:

```js
module.exports = require('@moxy/jest-config')({ react: true, enzyme: true ));
```